labgraph/devices/protocols/lsl/lsl_poller_node.py

```diff
@@ -0,0 +1,40 @@
+from pylsl import StreamInlet, resolve_stream
+from labgraph.graphs import AsyncPublisher, Config, Node, Topic, publisher
+from labgraph.util.logger import get_logger
+from labgraph.devices.protocols.lsl import LSLMessage
+
+import asyncio
+
+
+logger = get_logger(__name__)
+
+POLL_TIME = 0.1
```

Comment:
POLL_TIME is better to be put in PollerConfig; when user inherit the class, they shall be able to change the poll time

Reply:
Done